src/screens/CreatePost/index.js

```diff
@@ -706,7 +706,7 @@ const CreatePost = () => {
 
   React.useEffect(() => {
     const followType = params?.followType;
-    if (followType === 'incognito' || followType === undefined) {
+    if (followType === 'incognito') {
       setTypeUser(true);
     } else {
       setTypeUser(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made here is logical and makes sense. The `setTypeUser` function now only sets the user type to true when `followType` equals 'incognito'. Previously, it was also setting the user type to true when `followType` was undefined. This change seems to be more precise and avoids unnecessary state changes when `followType` is undefined.
```diff
-    if (followType === 'incognito' || followType === undefined) {
+    if (followType === 'incognito') {
       setTypeUser(true);
     } else {
       setTypeUser(false);
```
However, please ensure that there are no other parts of your code that rely on `typeUser` being set to true when `followType` is undefined, as this change could potentially break those parts.


<!-- This is an auto-generated comment by OSS CodeRabbit -->